From d7f5646de9ec990ed81489cc12d7942654bc017d Mon Sep 17 00:00:00 2001
From: Kam Nasim <kam.nasim@windriver.com>
Date: Fri, 23 Dec 2016 14:30:17 -0500
Subject: [PATCH] first meta patch to move python-keyring package from download
 tarball to srpm. Also updated to add tis patch versioning

---
 SPECS/python-keyring.spec | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/SPECS/python-keyring.spec b/SPECS/python-keyring.spec
index 8e51184..09cda6c 100644
--- a/SPECS/python-keyring.spec
+++ b/SPECS/python-keyring.spec
@@ -9,7 +9,7 @@
 
 Name:           python-keyring
 Version:        21.0.0
-Release:        1%{?dist}
+Release:        1.el8%{?_tis_dist}.%{tis_patch_ver}
 Summary:        Store and access your passwords safely
 License:        MIT and Python
 URL:            https://github.com/jaraco/keyring
@@ -43,7 +43,6 @@ Python keyring lib also provides following build-in keyrings.\
 Summary:        Python 2 library to access the system keyring service
 BuildRequires:  python2-devel
 BuildRequires:  python2-setuptools
-BuildRequires:  python2-setuptools_scm
 BuildRequires:  python2-keyczar
 BuildRequires:  python2-mock
 
@@ -72,7 +71,6 @@ Obsoletes:      %{name}-gnome < 0.6
 Summary:        Python 3 library to access the system keyring service
 BuildRequires:  python%{python3_pkgversion}-devel
 BuildRequires:  python%{python3_pkgversion}-setuptools
-BuildRequires:  python%{python3_pkgversion}-setuptools_scm
 %if 0%{?with_tests}
 %if 0%{?fedora} < 32
 BuildRequires:  python%{python3_pkgversion}-importlib-metadata
@@ -89,6 +87,9 @@ Requires:       python%{python3_pkgversion}-importlib-metadata
 
 %{?python_provide:%python_provide python%{python3_pkgversion}-keyring}
 
+Patch0: no_keyring_password.patch
+Patch8: keyring_path_change.patch
+
 %description -n python%{python3_pkgversion}-keyring
 %desc
 %endif
-- 
2.7.4

